airgapped/airgapped.go

```diff
@@ -179,18 +151,12 @@ func (am *Machine) ProcessOperation(operation client.Operation, storeOperation b
 			operation.ID, err)
 	}
 
-	if storeOperation {
+	if storeOperation && !strings.HasPrefix(string(operation.Type), "state_signing_") {
```

Comment:
There is at least two places where the same cumbersome check is performed. What do you think about creating an Operation.IsSigning operation method?